extensions/wikia/Email/templates/src/emailPreview.mustache

```diff
@@ -1,4 +1,6 @@
-{{#body}}{{{body}}}{{/body}}{{^body}}
+{{{body}}}
+
```

Comment:
Will this produce two blank lines in html when 'body' var is empty? Can't remember if Mustache trims it, probably it does :)